docs/docs/design/remote-funcitons.md

```diff
@@ -0,0 +1,85 @@
+# Remote Functions
+
+Remote functions are part of the DXOS extenisbility framework that enables developers to deploy long-running agents and invoke short-lived functions that have access to Spaces.
```

Comment:
i would avoid calling everything extensibility to help focus the words around just the frontend sdk. We want to be able to distinguish between user-space (apps, functions) and kernel-space (extensibility) domains, and would likely want to hold back on infrastructure extensibility while delivering user-space value early (i.e. this box runs your functions if you declare them like this. here's the tool to run it with ... .)

Reply:
which is distinct from "you can also extend this box to handle other scenarios we don't support" which is likely separable for a first hypothesis